gooddata-sdk/gooddata_sdk/catalog/workspace/declarative_model/workspace/logical_model/ldm.py

```diff
@@ -44,54 +48,100 @@ def load_from_disk(cls, workspace_folder: Path) -> CatalogDeclarativeModel:
 class CatalogDeclarativeLdm(Base):
     datasets: List[CatalogDeclarativeDataset] = attr.field(factory=list)
     date_instances: List[CatalogDeclarativeDateDataset] = attr.field(factory=list)
+    dataset_extensions: Optional[List[CatalogDeclarativeDatasetExtension]] = None
+
+    """
+    TODO: add support for dataset_extensions load and store
```

Comment:
Intentional TODO, or forgotten implementation?

Reply:
Forgotten. Thanks for the catch.